### DISCUSSION TITLE
Static build with XDebug for Mac

### DISCUSSION THREAD

**Discussion by andradedaniel** [php/frankenphp#848] [Q&A] [answered]
Hey everyone. 

Is it possible to create a static build with xdebug enabled in macos? 
How can I do that?

**Comment by withinboredom** [comment]
Why not just use a regular php build for debugging? Why use a static build?

**Comment by dunglas** [comment, marked as answer]
Xdebug isn't supported by static-php-cli, so it's not possible to use it with a static build: https://static-php.dev/en/guide/extension-notes.html#xdebug

You'll need a dynamic build.
